specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Subscriptions.json

```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/getVerificationId": {
+      "post": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "description": "Get the verification id of a subscription used for verifying custom domains",
+        "operationId": "GetVerificationId",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

please consider use the latest version "v5" for common types

Reply:
Version changed